x/ibc/02-client/keeper/client.go

```diff
@@ -72,14 +71,13 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 		return nil, sdkerrors.Wrapf(err, "cannot update client with ID %s", clientID)
 	}
 
+	consensusHeight := clientState.GetLatestHeight()
+
 	k.SetClientState(ctx, clientID, clientState)
 
 	// we don't set consensus state for localhost client
 	if header != nil && clientType != exported.Localhost {
 		k.SetClientConsensusState(ctx, clientID, header.GetHeight(), consensusState)
-		consensusHeight = consensusState.GetHeight()
-	} else {
-		consensusHeight = types.GetSelfHeight(ctx)
 	}
 
 	k.Logger(ctx).Info(fmt.Sprintf("client %s updated to height %d", clientID, clientState.GetLatestHeight()))
```

Comment:
Maybe this log should be changed since the update may not be updating the latest height

Reply:
yeah, should be consensusHeight